AzureBot/Dialogs/ActionDialog.cs

```diff
@@ -193,6 +193,43 @@ public class ActionDialog : LuisDialog<string>
             await this.ProcessVirtualMachineActionAsync(context, result, Operations.Shutdown, this.ShutdownVirtualMachineFormComplete);
         }
 
+        [LuisIntent("ListRunbooks")]
+        public async Task ListRunbooksAsync(IDialogContext context, LuisResult result)
+        {
+            var accessToken = await context.GetAccessToken(resourceId.Value);
+            if (string.IsNullOrEmpty(accessToken))
+            {
+                return;
+            }
+
+            var subscriptionId = context.GetSubscriptionId();
+
+            var automationAccounts = await new AzureRepository().ListRunbooksAsync(accessToken, subscriptionId);
+
+            if (automationAccounts.Any())

```

Comment:
We should detect if there is only one and then the output will look different versus more than one.